app/pb/api.proto

```diff
@@ -164,37 +164,44 @@ message User {
   string username = 2;
   string name = 3;
   string city = 4;
-  double verification = 5;
-  double community_standing = 6;
-  uint32 num_references = 7;
-  string gender = 8;
-  uint32 age = 9;
-  string color = 10; // web color, e.g. #643073, case doesn't matter
-  google.protobuf.Timestamp joined = 11; // not exact
-  google.protobuf.Timestamp last_active = 12; // not exact
-  HostingStatus hosting_status = 13;
-  string occupation = 14;
-  string about_me = 15;
-  string about_place = 16;
-  repeated string languages = 17;
-  repeated string countries_visited = 18;
-  repeated string countries_lived = 19;
-
-  FriendshipStatus friends = 20;
-  repeated MutualFriend mutual_friends = 21;
-  
-  google.protobuf.UInt32Value max_guests = 22;
-  google.protobuf.BoolValue multiple_groups = 23;
-  google.protobuf.BoolValue last_minute = 24;
-  google.protobuf.BoolValue accepts_pets = 25;
-  google.protobuf.BoolValue accepts_kids = 26;
-  google.protobuf.BoolValue wheelchair_accessible = 27;
-  SmokingLocation smoking_allowed = 28;
-  google.protobuf.StringValue sleeping_arrangement = 29;
-  google.protobuf.StringValue area = 30;
-  google.protobuf.StringValue house_rules = 31;
-
-  string avatar_url = 32;
+
+  // doubles are enough to describe lat/lng
+  // in EPSG4326
+  double lat = 5;
+  double lng = 6;
+  double radius = 7;
+
+  double verification = 8;
```

Comment:
Why did you change the number of existing protocol entries? It is not backwards compatible, and makes the diff unnecessary long.

Reply:
Fixed